queue.cpp

```diff
@@ -0,0 +1,118 @@
+#include <stdio.h>
+#include <stdlib.h>
+#define MAX 20
+
+//FUNCTION PROTOTYPES
+int QUEUE[MAX],REAR,FRONT;
+void DISPLAY(int []);
+void INSERT(int [],int);
+void DELETE(int []);
+void PEEK(int []);
+ 
+int main()
+{
+    int ITEM=0;
+    int choice=0;
+    int FRONT=REAR=-1; 

```

Comment:
here you are initializing ``REAR`` _(global)_, but creating a _local_ copy of ``FRONT`` (_global_ ``FRONT`` is uninitialized)